Eatery/Controllers/BRB/BRBViewController.swift

```diff
@@ -99,53 +127,62 @@ extension BRBViewController: BRBLoginViewControllerDelegate {
     func loginViewController(_ loginViewController: BRBLoginViewController,
                              didRequestLoginWithNetid netid: String,
                              password: String) {
-        connectionHandler.netid = netid
-        connectionHandler.password = password
-        connectionHandler.handleLogin()
+        accountManager.saveLoginInfo(loginInfo: LoginInfo(netid: netid, password: password))
+        accountManager.queryBRBData(netid: netid, password: password)
         
         loginViewController.isLoading = true
-        
-        requestStart = Date()
     }
     
 }
 
-extension BRBViewController: BRBConnectionDelegate {
+extension BRBViewController: BRBAccountManagerDelegate {
     
-    func retrievedSessionId(id: String) {
-        loginViewController.setShowErrorMessage(false, animated: true)
-        
-        NetworkManager.shared.getBRBAccountInfo(sessionId: id) { [weak self] (account, error) in
-            guard let self = self else {
-                return
-            }
-            
-            if let account = account {
-                self.loginViewController.isLoading = false
-                self.setState(.account(account))
-            } else {
-                self.loginFailed(with: error?.message ?? "")
-            }
+    func brbAccountManager(didQuery account: BRBAccount) {
+        if !loggedIn {
+            loginViewController.isLoading = false
         }
+        setState(.account(account))
+        activityIndicator.stopAnimating()
     }
     
-    func loginFailed(with error: String) {
-        loginViewController.errorDescription = error
-        loginViewController.setShowErrorMessage(true, animated: true)
-        loginViewController.isLoading = false
+    func brbAccountManager(didFailWith error: String) {
+        activityIndicator.stopAnimating()
+        if loggedIn {
+            showErrorAlert(error: error)
+        } else {
+            loginViewController.errorDescription = error
+            loginViewController.setShowErrorMessage(true, animated: true)
+            loginViewController.isLoading = false
+        }
     }
     
 }
 
+extension BRBViewController: BRBAccountViewControllerDelegate {
+
+    func brbAccountViewControllerDidRefresh() {
+        if !activityIndicator.isAnimating {
+            accountManager.queryBRBDataWithSavedLogin()
+            activityIndicator.startAnimating()
+        }
+    }
+
+}
+
 extension BRBViewController: AboutTableViewControllerDelegate {
-    
-    func aboutTableViewControllerDidLogoutUser(_ stvc: AboutTableViewController) {
+
+    func aboutTableViewControllerDidLogoutUser() {
         setState(.login)
-        
-        connectionHandler = BRBConnectionHandler()
-        connectionHandler.delegate = self
-        
+        navigationItem.title = "Account Info"
```

Comment:
we reference `"Account Info"`in `BRBViewController` multiple times, can we make it a class constant?